docs/user-guide/mvd-using.md

```diff
@@ -4,17 +4,18 @@ You can use the Zowe&trade; Application Framework to create application plug-ins
 
 ## Navigating the Zowe Desktop
 
-From the Zowe Desktop, you can access Zowe applications. 
+From the Zowe Desktop, you can access Zowe applications.
 
 ### Accessing the Zowe Desktop
 
-From a supported browser, open the Zowe Desktop at `https://{myhost}:{httpsPort}` or you can navigate to the direct Desktop URI at `https://{myhost}:{httpsPort}/ZLUX/plugins/org.zowe.zlux.bootstrap/web/`
+From a supported browser, open the Zowe Desktop at `https://zowe.externalDomains[0]:zowe.externalPort/zlux/ui/v1/` or you can navigate to the direct Desktop URI at `https://zowe.externalDomains[0]:zowe.externalPort/zlux/ui/v1/ZLUX/plugins/org.zowe.zlux.bootstrap/web/index.html`
     
 Where:
 
--   *myHost* is the host on which you are running the Zowe Application Server.
--   *httpsPort* is the value that was assigned to *node.https.port* in `server.json`.
-    For example, if you run the Zowe Application Server on host *myhost* and the value that is assigned to *node.https.port* in `server.json` is 12345, you would specify `https://myhost:12345/ZLUX/plugins/org.zowe.zlux.bootstrap/web/index.html`.
+-   *zowe.externalDomains[0]* is the host on which you are running the Zowe Application Server, its the 
+    value that was assigned in the zowe configuration file.
+-   *zowe.externalPort*  is the value that was assigned in the zowe configuration file.
```

Comment:
*zowe.externalPort* is the value of Gateway port that was assigned in the zowe configuration file.